BoxContentSDK/BoxContentSDK/OAuth2/BOXOAuth2Session.m

```diff
@@ -236,6 +247,9 @@ - (void)performRefreshTokenGrant:(NSString *)expiredAccessToken
                                                                                          body:POSTParams
                                                                                   queryParams:nil
                                                                                       session:self];
+    //Note: record the expired access token that initiated this refresh operation
+    //to detect if this refresh operation fails because we have acquired a newer access token
+    operation.accessToken = expiredAccessToken;
```

Comment:
Setting this doesn't cause us to send the `Authorization:` header in the request?

Reply:
we'll still have the expired access token in the auth header as before, but since it's token request, that header won't matter.